devstack/lib/nfp

```diff
@@ -243,84 +259,78 @@ function namespace_create {
+    gbp nat-pool-create --ip-version 4 --ip-pool $EXT_NET_CIDR --external-segment default --shared True default
+    gbp ep-create --external-segments default ext_connect
     gbp nsp-create --network-service-params type=ip_pool,name=vip_ip,value=nat_pool svc_mgmt_fip_policy
 }
 
-
 function create_advance_sharing_ptg {
-
-        gbp l3policy-create --ip-version 4 --ip-pool 121.0.0.0/20 --proxy-ip-pool=192.167.0.0/24 --subnet-prefix-length 20 advanced_services_sharing_l3p
-        gbp l2policy-create --l3-policy advanced_services_sharing_l3p advance_sharing_l2p
-        gbp group-create --l2-policy advance_sharing_l2p Advance_Sharing_PTG
+    gbp l3policy-create --ip-version 4 --ip-pool 121.0.0.0/20 --proxy-ip-pool=192.167.0.0/24 --subnet-prefix-length 20 advanced_services_sharing_l3p
+    gbp l2policy-create --l3-policy advanced_services_sharing_l3p advance_sharing_l2p
+    gbp group-create --l2-policy advance_sharing_l2p Advance_Sharing_PTG
 }
 
-
 function create_nfp_gbp_resources {
-   
     source $TOP_DIR/openrc neutron service
 
-    if [[ $DEVSTACK_MODE = base ]]; then
-        IMAGE_NAME_FLAT="reference_configurator_image"
+    if [[ $NFP_DEVSTACK_MODE = base ]]; then
+
+        IMAGE_NAME="reference_configurator_image"
         FLAVOR=m1.nfp-tiny
         gbp service-profile-create --servicetype LOADBALANCER --insertion-mode l3 --shared True --service-flavor service_vendor=haproxy,device_type=None --vendor NFP base_mode_lb
-        gbp service-profile-create --servicetype FIREWALL --insertion-mode l3 --shared True --service-flavor service_vendor=nfp,device_type=nova,image_name=$IMAGE_NAME_FLAT,flavor=$FLAVOR --vendor NFP base_mode_fw_vm
+        gbp service-profile-create --servicetype FIREWALL --insertion-mode l3 --shared True --service-flavor service_vendor=nfp,device_type=nova,image_name=$IMAGE_NAME,flavor=$FLAVOR --vendor NFP base_mode_fw_vm
+
     else
+
         gbp service-profile-create --servicetype LOADBALANCER --insertion-mode l3 --shared True --service-flavor service_vendor=haproxy,device_type=nova --vendor NFP lb_profile
-        gbp service-profile-create --shared True --vendor NFP --servicetype LOADBALANCERV2 --service-flavor service_vendor=haproxy_lbaasv2,device_type=nova,flavor=m1.small --insertion-mode l3 lbv2_profile
+        gbp service-profile-create --servicetype LOADBALANCERV2 --insertion-mode l3 --shared True --service-flavor service_vendor=haproxy_lbaasv2,device_type=nova,flavor=m1.small --vendor NFP lbv2_profile
         gbp service-profile-create --servicetype FIREWALL --insertion-mode l3 --shared True --service-flavor service_vendor=vyos,device_type=nova --vendor NFP vyos_fw_profile
         gbp service-profile-create --servicetype VPN --insertion-mode l3 --shared True --service-flavor service_vendor=vyos,device_type=nova --vendor NFP vpn_profile
 
-        if [[ $DEVSTACK_MODE = enterprise ]]; then
+        if [[ $NFP_DEVSTACK_MODE = enterprise ]]; then
```

Comment:
Right now, some parts of this file are having this check for enterprise mode. But we didn't submit them. We have to work on separating this file from the enterprise mode changes.